ipmi-config/ipmi-config-category-core-lan-conf-section.c

```diff
@@ -36,6 +37,7 @@
 #include "freeipmi-portability.h"
 #include "pstdout.h"
 
+#define BMC_MAXIPV6ADDRLEN 40
```

Comment:
Lets go w/ 45 instead of 40, as that's what I use in other parts of code.  B/c this is possible in other areas:

```ABCD:ABCD:ABCD:ABCD:ABCD:ABCD:192.168.158.190```